.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+// These are GitHub Codespaces specific settings.
+{
+  "name": "Wasp Todo App Tutorial",
+  "customizations": {
+    "vscode": {
+      "extensions": ["wasp-lang.wasp", "PKief.material-icon-theme"],
+      "settings": {
+        "workbench.iconTheme": "material-icon-theme"
+      }
+    }
+  },
+  "containerEnv": {
+    "WASP_TELEMETRY_CONTEXT": "codespaces"
+  },
+  "forwardPorts": [3000, 3001],
+  "portsAttributes": {
+    "3000": {
+      "label": "React-Vite Client"
+    },
+    "3001": {
+      "label": "NodeJS Server"
+    }
+  },
+  "postCreateCommand": "curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s",
+  "postStartCommand": "printf 'export WASP_WEB_CLIENT_URL=https://$CODESPACE_NAME-3000.app.github.dev\n' >> $HOME/.bashrc && printf 'export REACT_APP_API_URL=https://$CODESPACE_NAME-3001.app.github.dev\n' >> $HOME/.bashrc && echo 'gh codespace ports visibility 3000:public -c $CODESPACE_NAME\n' >> $HOME/.bashrc && echo 'gh codespace ports visibility 3001:public -c $CODESPACE_NAME\n' >> $HOME/.bashrc",
```

Comment:
Sounds like this one runs on every start of codespace? If so, we will be adding this stuff to .bashrc again and again on every start, won't we? Won't .bashrc keep growing and growing then, if its content is being persisted? I don't know how codespaces work nor what postStartCommand does so I might be wrong, but it seemed like this might be the case.

Reply:
You're right. I switched the wasp install to `onCreateCommand` and the env var exporting to `postCreateCommand`